ui/tests/acceptance/secrets/backend/secret-engine-list-view.js

```diff
@@ -33,7 +33,21 @@ module('Acceptance | secret-engine list view', function (hooks) {
     return login();
   });
 
-  test('it allows you to disable an engine', async function (assert) {
+  // TEST 1:
```

Comment:
So for this PR, I actually think we need to write these tests. I know I didn't have them in the spike, but moving the code into a component was to set these up and I think it's best practice to put them in now especially as you may get pulled into a different direction soon. In other words, let's not merge this until these tests are done.

I'm more than happy to help write these out. We can set up a pairing session to do so.